packages/terra-form/src/components/Input.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Input.scss';
+
+const propTypes = {
+  /**
+   * The defaultValue of the input field. Use this to create an uncontrolled input
+   */
+  defaultValue: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
+  /**
+   * Name of the input
+   */
+  name: PropTypes.string,
+  /**
+   * Functional to be called when the input is changed
+   */
+  onChange: PropTypes.func,
+  /**
+   * Whether the input is required or not
+   */
+  required: PropTypes.bool,
+  /**
+   * The value of the input field. Use this to create a controlled input
+   */
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
+};
+
+const defaultProps = {
+  defaultValue: undefined,
+  name: null,
+  onChange: () => {},
```

Comment:
We are inconsistent between packages on how to handle default onChange/onClick/onKeyDown functions. Button has no default defined, ActionHeader defaults to null, Table & List defaults to undefined, and here it defaults to an empty function. Do we need to use a standard?